Sources/Core/JSFileRenderer.swift

```diff
@@ -30,7 +30,9 @@ extension JSFileRenderer {
     }
 
     var properties: [(Parameter, SchemaObjectProperty)] {
-        return self.rootSchema.properties.map { $0 }
+        return self.rootSchema.properties.map { $0 }.sorted(by: { (obj1, obj2) -> Bool in
```

Comment:
You could also do it shorter: `self.rootSchema.properties.map {$0}.sorted(by: { $0.0 < $1.0})`